boost.sh

```diff
@@ -199,6 +199,9 @@ OPTIONS:
     --no-framework
         Do not create the framework.
 
+    --universal
+        Create universal static library for iOS.
```

Comment:
I don't think it's necessary to specify that the library is static. This script _only_ outputs static libraries. iOS & tvOS _must_ use static libraries or frameworks - you cannot link shared libraries. Also, as I mention below, we'll want to add this for tvOS as well, so we shouldn't specify iOS here. So, maybe this description should read "Create universal FAT binary" instead.